__tests__/app.js

```diff
@@ -30,7 +30,8 @@ describe('node:app', () => {
         authorEmail: 'hi@yeoman.io',
         authorUrl: 'http://yeoman.io',
         keywords: ['foo', 'bar'],
-        includeCoveralls: true
+        includeCoveralls: true,
+        nodejsVersion: 'v10.4.1,v10'
```

Comment:
Let's just use `node` instead of `nodejs` for all the namings. (there's a few other spot to also update)